blocks/address/src/address-card.tsx

```diff
@@ -0,0 +1,340 @@
+                  height: "fit-content",
+                  borderRadius: "50%",
+                  minWidth: "unset",
+                  minHeight: "unset",
+                  padding: 1,
+                }}
+              >
+                <FontAwesomeIcon icon={faCopy} sx={{ fontSize: 12 }} />
+              </Button>
+            ) : null}
+          </Box>
+        </Stack>
+
+        <Stack gap={1.5}>
+          <MapButton href={googleMapsUrl}>
+            <GoogleIcon sx={{ fontSize: 18, mr: 1 }} />
+            Open in Google Maps
+          </MapButton>
+          <MapButton href={appleMapsUrl}>
+            <AppleIcon sx={{ fontSize: 18, mr: 1 }} />
+            Open in Apple Maps
+          </MapButton>
+        </Stack>
+
+        {description || editingDescription || readonly ? (
+          <EditableField
+            value={descriptionValue}
+            onChange={(event) => setDescriptionValue(event.target.value)}
+            onBlur={(event) => {
+              setEditingDescription(false);
+              updateDescription(event.target.value);
+            }}
+            placeholder="Enter description"
+            iconSize="14px"
+            inputProps={{
+              sx: {
+                fontWeight: 500,
+                fontSize: 14,
+                lineHeight: 1.3,
+                letterSpacing: "-0.02em",
+                color: theme.palette.gray[90],
+              },
+            }}
+            readonly={readonly}
+          />
+        ) : (
+          <Typography
+            onClick={() => {
+              setEditingDescription(true);
+            }}
+            sx={{
+              fontWeight: 500,
+              fontSize: 14,
+              lineHeight: 1.3,
+              letterSpacing: "-0.02em",
+              color: theme.palette.gray[50],
+            }}
+          >
+            Click here to add a description or more detailed information
+            <FontAwesomeIcon icon={faPenToSquare} sx={{ ml: 1 }} />
+          </Typography>
+        )}
+      </Stack>
+
+      <Box
+        sx={({ palette }) => ({
+          display: "flex",
+          alignItems: "center",
+          justifyContent: "center",
+          position: "relative",
+          background: palette.gray[10],
+          borderLeft: `1px solid ${palette.gray[20]}`,
+          width: 500,
+          [theme.breakpoints.down("md")]: {
+            width: 1,
+            height: 300,
+          },
+        })}
+      >
+        {mapUrl ? (
+          <Box
+            sx={{
+              width: 1,
+              height: 1,
+              background: `url(${mapUrl}) no-repeat`,
+              backgroundSize: "cover",
+            }}
+          >
+            <Fade in={hovered}>
+              <Stack
+                sx={{
+                  display: "inline-flex",
+                  position: "absolute",
+                  top: 13,
+                  left: 13,
+                }}
+              >
+                <Button
```

Comment:
This should not appear in `readonly`

Reply:
Hid it in readonly mode in 38c4a19ea9d64daea84706f708acccffd3e7ebc9

Reply:
We can do that, although it would trigger more API requests than pageviews. I would default to no, but it could be added later as a configuration option for the block.